src/components/App/App.spec.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from './App';
+import Calculator from '../Calculator/Calculator';
```

Comment:
We can mock our dependencies to increase our test performance. Here is a good example. We want to verify that the App component returns the `Calculator` component, but we don't need to import the component at all. We just need to mock it, and test for the mocks instance. We can do this like this: 

```suggestion
jest.mock('../Calculator/Calculator', () => 'Calculator');
```